x/dsmr/certificate.go

```diff
@@ -1,68 +1,48 @@
 // Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
+//go:generate go run github.com/StephenButtolph/canoto/canoto --concurrent=false $GOFILE
+
 package dsmr
 
 import (
 	"context"
-	"errors"
 	"fmt"
 
-	"github.com/ava-labs/avalanchego/codec/linearcodec"
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow/validators"
-	"github.com/ava-labs/avalanchego/utils/units"
-	"github.com/ava-labs/avalanchego/utils/wrappers"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
 	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
-
-	"github.com/ava-labs/hypersdk/codec"
-
-	acodec "github.com/ava-labs/avalanchego/codec"
-)
-
-const (
-	CodecVersion = 0
-
-	MaxMessageSize = units.KiB
```

Comment:
Was `MaxMessageSize` used to actually enforce something important? Or was this just used to conform to the avalanchego codec?